### ISSUE TITLE
default.yaml is always moved to ~/.config/ibus/rime/trash

### ISSUE THREAD

**Issue by ChandelierX** [rime/ibus-rime#44]
After recent update, or after I installed some external fonts (simhei and simsun) in my Manjaro Linux (based on Arch), 
my ibus-rime doesn't support my `default.custom.yaml` any more (it is simply ignored, and doesn't work).

Besides, my `default.yaml` is auto-removed to `~/.config/ibus/rime/trash/` each time after my deploy.

Thus, it doesn't list my double_pinyin_flypy schema, but only lists luna_pinyin and some other default input schema.

Here's the log:
```
$ cat /tmp/rime.ibus.WARNING

Log file created at: 2018/05/27 23:19:21
Running on machine: chandelier-pc
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
E0527 23:19:21.606014 15032 config_data.cc:41] failed to save config to stream.
E0527 23:27:23.406353 16488 config_data.cc:41] failed to save config to stream.
```

What Can I do to invoke double_pinyin_flypy, to make my `default.yaml` work?

**Comment by ChandelierX** [comment]
Ok, I managed to solve this problem by simply depriving the `execute attribute` of the `~/.config/ibus/rime/trash` directory.
By this means, the `default.yaml` won't be moved into trash.
But why this delete action happens?

**Comment by lotem** [comment]
Files are moved to `trash/` if they are not newer (older or same) than the installed version in `/usr/share/rime-data/`.

Your customization to files that are updated via package manager should be put in patch files like `default.custom.yaml`. It should work. If it doesn't, look for reported errors in log files.

**Comment by ChandelierX** [comment]
Thanks for your reply, it's informative and helpful.
But as for `default.custom.yaml`, no, it still doesn't work, I am looking for errors in log.

**Comment by ChandelierX** [comment]
Solved. Yes I can be solved by change `config_version: "1.0"`
